import "protocol.proto";

package OstProto;
message Vlan {
	// VLAN presence/absence
	optional bool is_override_tpid = 1;

	// VLAN values
	optional uint32	tpid = 2;
	optional uint32	vlan_tag = 3; // includes prio, cfi and vlanid
}

extend Stream {
	optional Vlan vlan = 126;
}
